### ISSUE TITLE
Agryll installation failure

### ISSUE THREAD

**Issue by Agilulfo** [eoyilmaz/displaycal-py3#291]
I've manged to checkout and install DisplayCal from this repo.
When the start I get a dialog warning me that Agryll is not present in my system and it proposes me to install it for me.
However the download fails with what seems to be a 404 response from the server where the installer tries to pull it.
```
Ready.
Setting up scripting host at 127.0.0.1:15411
Check for application update...
DisplayCAL is up-to-date.
┌──────────────────────────────────────────────────────────────────────────────┐
│ Download ArgyllCMS 2.3.1                                                     │
└──────────────────────────────────────────────────────────────────────────────┘
-> Download
Connecting to https://displaycal.net:443...
Warning: Missing bitmap 'theme/icons/256x256/displaycal'
Warning: Missing bitmap 'theme/icons/48x48/displaycal'
Warning: Missing bitmap 'theme/icons/32x32/displaycal'
Warning: Missing bitmap 'theme/icons/16x16/displaycal'

┌──────────────────────────────────────────────────────────────────────────────┐
│ HTTP Error 400: Bad Request                                                  │
└──────────────────────────────────────────────────────────────────────────────┘

```

might be worth to update the source or replace the installation feature with a link to a guide on how to install agryll.

**Comment by Agilulfo** [comment]
P.S. my workaround was to download Agryll from [here](https://www.argyllcms.com/) unzip it and then restart DisplayCal and select it manually.

**Comment by eoyilmaz** [comment]
Let's solve this...

**Comment by eoyilmaz** [comment]
Fixed in 1ef1898 ... Although I didn't like the fix I introduced. But we don't have access to `displaycal.net` and we cannot update the required files stored there. So we are now donwloading the ArgyllCMS executables from `argyllcms.com` but the version number is temporarily hardcoded to `3.1.0`... I'll create a feature ticket to implement a better way of checking newer versions of ArgyllCMS. Closing this...
